toml/src/main/kotlin/org/rust/toml/crates/local/CratesLocalIndexService.kt

```diff
@@ -134,7 +136,7 @@ class CratesLocalIndexService : PersistentStateComponent<CratesLocalIndexState>,
         return crateNames
     }
 
-    private fun updateIfNeeded() {
+    fun updateIfNeeded() {
```

Comment:
We usually mark such methods with `com.google.common.annotations.VisibleForTesting` annotation. I believe in the future kotlin plugin will have an inspection to detect usages of marked methods in inappropriate places. Java plugin has such inspection